save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/ReactorUtils.kt

```diff
@@ -189,7 +189,7 @@ fun <T : Any> blockingToMono(supplier: () -> T?): Mono<T> = supplier.toMono()
  * @param supplier blocking operation like JDBC
  * @return [Flux] from result of blocking operation [List] of [T]
  */
-fun <T> blockingToFlux(supplier: () -> Iterable<T>): Flux<T> = blockingToMono(supplier).flatMapIterable { it }
+fun <T> blockingToFlux(supplier: () -> Iterable<T>?): Flux<T> = blockingToMono(supplier).flatMapIterable { it }
```

Comment:
Where exactly do we need this signature change?

A nullable iterable should normally be replaced with an empty iterable instead.